frontend/src/components/Lobby/Lobby.js

```diff
@@ -34,6 +33,10 @@ const Lobby = () => {
           history.push("/game");
           break;
         case "players":
+          response.players.splice(
+            response.players.indexOf(Cookies.get("username")),
+            1
+          );
           setPlayersList([...makePlayersList(response.players)]);
```

Comment:
`splice` mutates the array that we might want to use later, so IMO use `filter` instead like so:
```JavaScript
          case "players":
            setPlayersList(
              [...makePlayersList(
                 response.players.filter(username => username != Cookies.get("username")
              )]
            );
```

Reply:
`filter` however is less efficient since we are iterating through the whole array, so you might want to use splice but on a copy of the array, if you care about _p e r f o r m a n c e_